src/renderer.js

```diff
@@ -110,6 +110,11 @@ export default class Renderer {
   }
 
   codespan(text) {
+    if (this.options.codeSpanHighlighted) {
+      const defaultInlineLanguage = 'markup'
+      return `<code class="${this.options
+        .langPrefix}${defaultInlineLanguage}">${text}</code>`
+    }
     return `<code>${text}</code>`
```

Comment:
why not process the code with `options.highlight` function?

Reply:
Yes, good point. I thought about that. But I ended up adding the default class (and as an option), because on the "inline code spans", there is no way (AFAIK) to specify the inferred lang. And using prismjs, `lang` is a required parameter.

`options.highlight` is not prismjs-only related. So I guess, that [line 114](https://github.com/egoist/md/pull/13/files/63acf24bf723b49876414861a5eb2527b43a60bf#diff-5326222f837d36fd2bff476584c07621R114) could be extracted to the options as well?

Reply:
I've just realized, `markup` could be the `lang`. Let me try that and submit The PR again.

Reply:
BTW, this is the use case I was facing:
https://codesandbox.io/s/yjn2vxqn2z

If you have any suggestion, please, share them with me ;)

Reply:
I'm a little bit stuck right now. :/

Reply:
I now have understood what you suggested. But I think [renderer](https://marked.js.org/#/USING_ADVANCED.md#options) is the correct option for extensibility. I'm going to try it on my own. Thanks, anyway!